README.md

```diff
@@ -1,51 +1,37 @@
 
+Specifically, we're building on the **Contrast Consistent Search** (CCS) method described in the paper [Discovering Latent Knowledge in Language Models Without Supervision](https://arxiv.org/abs/2212.03827) by Burns et al. (2022). In CCS, we search for features in the hidden states of a language model which satisfy certain logical consistency requirements. It turns out that these features are often useful for question-answering and text classification tasks, even though the features are trained without labels.
 
 ### Quick **Start**
 
-First install the package with `pip install -e .` in the root directory, or `pip install -e .[dev]` if you'd like to contribute to the project (see **Development** section below). This should install all the necessary dependencies.
+Our code is based on [PyTorch](http://pytorch.org) and [Huggingface Transformers](https://huggingface.co/docs/transformers/index). We test the code on Python 3.9 and 3.10.
 
-For a quick test: You can look into and run generate.sh and evaluate.sh (**Warning:** They are in the package elk itself right now. Will be changed):
-
-```bash
-cd elk
-sh generate.sh
-sh train_evaluate.sh
-```
-
-Furthermore:
+First install the package with `pip install -e .` in the root directory, or `pip install -e .[dev]` if you'd like to contribute to the project (see **Development** section below). This should install all the necessary dependencies.
 
-1. To generate the hidden states for one model `mdl` and all datasets, `cd elk` and then run
+1. To extract the hidden states for one model `model` and the dataset `dataset`, run:
 
 ```bash
-python extraction_main.py --model deberta-v2-xxlarge-mnli --datasets imdb --prefix normal --device cuda --num-data 1000
+elk extract microsoft/deberta-v2-xxlarge-mnli imdb --max-examples 1000
 ```
 
-To test `deberta-v2-xxlarge-mnli` with the misleading prefix, and only the `imdb` and `amazon-polarity` datasets, while printing extra information, run:
-
-The name of prefix can be found in `./extraction/construct_prompts.py`. This command will save hidden states to `extraction_results` and will save zero-shot accuracy to `extraction_results/extraction_results.csv`.
+To test `microsoft/deberta-v2-xxlarge-mnli` with the misleading prefix, and only the `imdb` and `amazon_polarity` datasets, while printing extra information, run:
 
-1. To train a ccs model and a logistic regression model
+1. To train a CCS model and a logistic regression model
 
 ```bash
-python train.py --model deberta-v2-xxlarge-mnli --prefix normal --dataset imdb --num-data 1000
+elk train microsoft/deberta-v2-xxlarge-mnli imdb
```

Comment:
Is this compatible with the "name" argparse? It doesn't look like it does, but I'm not familiar enough with argparse to be sure

Reply:
Yeah this needs to be changed